packages/getting-started/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) IBM Corp. 2017. All Rights Reserved.
+Node module: @loopback/getting-started
```

Comment:
Can we name the package as `@loopback/example-getting-started` to be consistent with `@loopback/example-codehub`?